activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -252,6 +259,19 @@ def load_seed
         end
       end
 
+      def self.run_cmd(cmd, args, action)
+        fail "Invalid database command - #{cmd}" unless valid_bin? cmd
+        command = args.dup.unshift([cmd, cmd])
```

Comment:
I'm not sure I understand why we need to `unshift` `[cmd, cmd]`. According to the docs, `Kernel.system(cmd, *args)` should do the same, no?

```
commandline                 : command line string which is passed to the standard shell
cmdname, arg1, ...          : command name and one or more arguments (no shell)
[cmdname, argv0], arg1, ... : command name, argv[0] and zero or more arguments (no shell)
```

http://ruby-doc.org/core-2.2.0/Kernel.html#method-i-system